### DISCUSSION TITLE
[Feature Request] Carousel!!

### DISCUSSION THREAD

**Discussion by Oolix** [heroui-inc/heroui#1973] [Feature requests]
### Is your feature request related to a problem? Please describe.

N/A

### Describe the solution you'd like

I'd like a Carousel component to be implemented I think that would be an excellent addition to this robust library. Trying to find another library that's lightweight and works well with Nextjs/TS is a pain.

### Describe alternatives you've considered

https://www.npmjs.com/package/react-responsive-carousel

**Comment by wingkwong** [comment]
let's track in #4678
